packages/client/test/sync/skeleton.spec.ts

```diff
@@ -159,7 +159,7 @@ tape('[Skeleton]', async (t) => {
     for (const [testCaseIndex, testCase] of testCases.entries()) {
       const config = new Config({ common, transports: [] })
       const chain = new Chain({ config })
-      const metaDB = level()
+      const metaDB: any = new MemoryLevel()
```

Comment:
Can't we remove these `any` type casts now as well since you changed the types over to `abstract-level`? I checked locally and Typescript seems to be happy with it without the `:any` casting.

Reply:
This is my last nit.  This is fantastic work so thank you so much!  I know this has been a long slog.